index.js

```diff
@@ -0,0 +1,32 @@
+const { execSync } = require('child_process');
+// stderr is sent to stderr of parent process
+const AWS = require("aws-sdk")
+const fs = require("fs")
+// you can set options.stdio if you want it to go elsewhere
+//
+//Expect payload on the event object
+exports.handler =  async function(event, context) {
+  process.env[`PATH`] = process.env[`PATH`] + `:` + process.env[`LAMBDA_TASK_ROOT`]
+  //console.log("EVENT: \n" + JSON.stringify(event, null, 2))
+  //return context.logStreamName
+	// Since this is not parameterized its is very dangerous
```

Comment:
?

Reply:
Oh, because it's a shell injection vuln waiting to happen. Got it.

Reply:
Just realized I can get rid of this and just use a javascript function

Reply:
done